controllers/keda/hpa.go

```diff
@@ -29,7 +29,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
-	kedav1alpha1 "github.com/kedacore/keda/v2/apis/keda/v1alpha1"
+	"github.com/kedacore/keda/v2/apis/keda/v1alpha1"
```

Comment:
why this change? TBH I prefer the original, so we have consistent naming of this dependency across the codebase.